app/models/health/release_form.rb

```diff
@@ -37,7 +37,7 @@ class ReleaseForm < HealthBase
     scope :recent, -> { order(signature_on: :desc).limit(1) }
     scope :reviewed, -> { where.not(reviewed_by_id: nil) }
     scope :valid, -> do
-      parent_ids = Health::ReleaseFormFile.where.not(parent_id: nil).pluck(:parent_id)
+      parent_ids = Health::ReleaseFormFile.where.not(parent_id: nil).select(:parent_id)
```

Comment:
Pretty sure this is safe, but @awisema can you confirm these parent IDs are going to be used in the same database?  With so many cross-database situations, just want to double check a `select` is ok here.

Reply:
I have confirmed that the parent of a `Health::ReleaseFormFile` is the `Health::ReleaseForm` it is uploaded against -- so, there should be no x-db issue here.

Reply:
Thanks